### ISSUE TITLE
Question about destination .

### ISSUE THREAD

**Issue by maccam04** [sellmair/kompass#16]
Hi , currently im using your library its great and ease to implement and may i know how to identify which is destination is currently is active? or is there a currently function to that please let me know thanks.

**Comment by sellmair** [comment]
I am so sorry, that I overlooked your question!!!! 
Had a few very busy weeks at work =/

So there is currently nothing to retrieve the current destination, but it would be easy to implement this 🤔 

Let me know if you still need this feature.

Also I tested Kompass a lot during development for a pretty huge customer and everything looks fine, that's why I plan to re-write a lot of Kompass to get it a pretty solid lib for many use cases. 💪 

I also plan to add non-fragment support, which I consider much more powerful! 🤔
